### ISSUE TITLE
Make Waterfall Green Again

### ISSUE THREAD

**Issue by nox** [servo/servo#11365]
```
warning: unused variable: `id`, #[warn(unused_variables)] on by default
  --> /Users/nox/src/servo/components/script/dom/file.rs:55:13
55 |>         let id = selected.id;
   |>             ^^
```

**Comment by izgzhen** [comment]
Ah my bad. I leave a `// FIXME: fix this after PR #11221 is landed` just before the line. But currently #11221 is still in pending.

**Comment by jdm** [comment]
For those not in the know: "waterfall" refers to http://build.servo.org/waterfall which shows orange jobs due to warnings; green ones would have no warnings. The goal of this issue is to fix the warning that is making it orange.

**Comment by mbrubeck** [comment]
Fixed by #11368
